dask/dataframe/tests/test_multi.py

```diff
@@ -484,8 +556,9 @@ def test_join_by_index_patterns(how, shuffle):
             """
 
 
-@pytest.mark.parametrize('how', ['inner', 'outer', 'left', 'right'])
-@pytest.mark.parametrize('shuffle', ['disk', 'tasks'])
+@pytest.mark.parametrize(['how', 'shuffle'],
+                         product(['inner', 'left', 'right', 'outer'],
+                                 ['disk', 'tasks']))
```

Comment:
Changed to see match how parametrize is used elsewhere, and to see if there was any overhead to stacking them (the multi tests are by far the slowest in dataframe). Found no change, but never reverted.